Library/Homebrew/cmd/commands.rb

```diff
@@ -20,14 +20,14 @@ class CommandsCommand < Command
       optional_arg :formulae1
```

Comment:
Instead of having two different methods for optional and mandatory trailing arguments, let's just have one (`argument :car`) and a modifier in the form of a parameter, like `optional: true`. So, something like this:

```ruby
argument :formulae1
argument :car, optional: true
```

That way the mandatory arguments can be implicit as well.